xml/admin_ceph_upgrade.xml

```diff
@@ -935,6 +1027,152 @@ warning: /etc/ganesha/ganesha.conf saved as /etc/ganesha/ganesha.conf.rpmsave</s
     </step>
    </procedure>
   </sect2>
+
+  <sect2 xml:id="upgrade-mds">
+   <title>Upgrading the &mds;</title>
+   <para>
+    Unlike MONs, MGRs and OSDs, Metadata servers cannot be adopted in-place.
+    Instead, you need to redeploy them in containers using the &ceph; orchestrator.
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Run <command>ceph fs ls</command> to obtain the name of your file system, for example:
+     </para>
+<screen>
+&prompt.cephuser;ceph fs ls
+name: cephfs, metadata pool: cephfs_metadata, data pools: [cephfs_data ]
+</screen>
+    </step>
+    <step>
+     <para>
+      Create a new service specification file <filename>mds.yml</filename> as
+      described in <xref linkend="deploy-cephadm-day2-service-mds"/> by using
+      the file system name as the <option>service_id</option> and specifying
+      the hosts that will run the MDS daemons. For example:
+     </para>
+<screen>
+service_type: mds
+service_id: cephfs
+placement:
+  hosts:
+  - ses-min1
+  - ses-min2
+  - ses-min3
+</screen>
+    </step>
+    <step>
+     <para>
+      Run <command>ceph orch apply -i mds.yml</command> to apply the service
+      specification and start the MDS daemons.
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="upgrade-igw">
+   <title>Upgrading the &igw;</title>
+   <para>
+    &igw;s cannot be adopted in-place. Instead, you need to redeploy them in
```

Comment:
Okay, same comment as above - but I'm picking up what you're putting down. I think this can be removed.